### PR TITLE
Allow access to keras.utils.model_to_dot API

### PR DESCRIPTION
**Pull request by sibyjackgrove**
`model_to_dot` is not currently accessible from TensorFlow Keras. So visualizing the graph directly using commands  like` SVG(model_to_dot(model).create(prog='dot', format='svg'))` is not possible. 

Add `@tf_export('keras.utils.model_to_dot')` in  vis_utils.py

Add `from tensorflow.python.keras._impl.keras.utils.vis_utils import model_to_dot` in 
'tensorflow/python/keras/_impl/keras/utils/__init__.py'

Add `from tensorflow.python.keras._impl.keras.utils.vis_utils import model_to_dot` in 
'tensorflow/python/keras/utils//__init__.py'

### CODE REVIEW

**Comment by sibyjackgrove** [comment]
@fchollet Can you take a look at this, please?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jhseu: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by fchollet — APPROVED** [review]
LGTM

Sorry for the delayed review.

**Comment by sibyjackgrove** [comment]
Thank you @fchollet @jhseu

**Comment by sibyjackgrove** [comment]
It seems to be failing on Python2 for target //tensorflow/tools/api/tests:api_compatibility_test
Any suggestions?

**Comment by jhseu** [comment]
Can you run the test locally? It'll show the message for how to update the goldens for API changes.

**Comment by sibyjackgrove** [comment]
@jhseu how do I run test locally? I have applied the proposed changes on the TensorFlow installation in my local machine. The `model_to_dot `API works as expected and I don't get any error messages.

**Comment by jhseu** [comment]
To be specific, the failing test here: `//tensorflow/tools/api/tests:api_compatibility_test`. We keep a list of APIs available to users, and that test failure will explain how to update the golden files.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jhseu: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by martinwicke** [comment]
Can you fix the conflicts? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by sibyjackgrove** [comment]
Since the` /_impl/` folder was removed I think this PR is not valid anymore. I will close and create a new PR. Though still no idea how to resolve the API conflicts.
